README.md

```diff
@@ -0,0 +1,310 @@
+<!-- markdownlint-disable MD033 -->
+<!-- x-hide-in-docs-start -->
+<p align="center">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/open-feature/community/0e23508c163a6a1ac8c0ced3e4bd78faafe627c7/assets/logo/horizontal/white/openfeature-horizontal-white.svg" />
+    <img align="center" alt="OpenFeature Logo" src="https://raw.githubusercontent.com/open-feature/community/0e23508c163a6a1ac8c0ced3e4bd78faafe627c7/assets/logo/horizontal/black/openfeature-horizontal-black.svg" />
+  </picture>
+</p>
+
+<h2 align="center">OpenFeature Python SDK</h2>
+
+<!-- x-hide-in-docs-end -->
+<!-- The 'github-badges' class is used in the docs -->
+<p align="center" class="github-badges">
+
+  <a href="https://github.com/open-feature/spec/releases/tag/v0.3.0">
+    <img alt="Specification" src="https://img.shields.io/static/v1?label=Specification&message=v0.3.0&color=red&style=for-the-badge" />
+  </a>
+
+  <!-- x-release-please-start-version -->
+
+  <a href="https://github.com/open-feature/python-sdk/releases/tag/v0.3.1">
+    <img alt="Latest version" src="https://img.shields.io/static/v1?label=release&message=v0.3.1&color=blue&style=for-the-badge" />
+  </a>
+
+  <!-- x-release-please-end -->
+  <br/>
+  <a href="https://github.com/open-feature/python-sdk/actions/workflows/merge.yml">
+    <img alt="Build status" src="https://github.com/open-feature/python-sdk/actions/workflows/build.yml/badge.svg" />
+  </a>
+
+  <a href="https://codecov.io/gh/open-feature/python-sdk">
+    <img alt="Codecov" src="https://codecov.io/gh/open-feature/python-sdk/branch/main/graph/badge.svg?token=FQ1I444HB3" />
+  </a>
+
+  <a href="https://www.python.org/downloads/">
+    <img alt="Min python version" src="https://img.shields.io/badge/python->=3.8-blue.svg" />
+  </a>
+
+  <a href="https://www.repostatus.org/#wip">
+    <img alt="Repo status" src="https://www.repostatus.org/badges/latest/wip.svg" />
+  </a>
+</p>
+<!-- x-hide-in-docs-start -->
+
+[OpenFeature](https://openfeature.dev) is an open specification that provides a vendor-agnostic, community-driven API for feature flagging that works with your favorite feature flag management tool.
+
+<!-- x-hide-in-docs-end -->
+
+## 🚀 Quick start
+
+### Requirements
+
+- Python 3.8+
+
+### Install
+
+<!---x-release-please-start-version-->
+
+#### Pip install
+
+```bash
+pip install openfeature-sdk==0.3.1
+```
+
+#### requirements.txt
+
+```bash
+openfeature-sdk==0.3.1
+```
+
+```python
+pip install -r requirements.txt
+```
+
+<!---x-release-please-end-->
+
+### Usage
+
+```python
+from openfeature import api
+from openfeature.provider.in_memory_provider import InMemoryFlag, InMemoryProvider
+
+# flags defined in memory
+my_flags = {
+  "v2_enabled": InMemoryFlag("on", {"on": True, "off": False})
+}
```

Comment:
This configuration requires [this change](https://github.com/open-feature/python-sdk/pull/228) to be released.

Reply:
I'll release `0.4.0` right after merging this.